src/Ninject.Web.AspNetCore.Test/Ninject.Web.AspNetCore.Test.csproj

```diff
@@ -0,0 +1,65 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;netcoreapp2.2</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\Ninject.Web.AspNetCore.snk</AssemblyOriginatorKeyFile>
+    <Configurations>Debug;Release22;Release31</Configurations>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release22'">
+    <TargetFrameworks>netcoreapp2.2;netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release31'">
+    <TargetFrameworks>netcoreapp3.1;netcoreapp3.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="5.9.0" />
+    <PackageReference Include="Moq" Version="4.15.1" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.console" Version="2.4.1">
```

Comment:
Thanks, I missed the Microsoft.NET.Test.Sdk and therefore it could not run it in visual studio and also not on command line. I had first added xunit.runner.visualstudio but that one alone did not work... Thanks a lot for the hint :-)